README.md

```diff
@@ -39,7 +39,7 @@ Trapheus can be scheduled using a cloudwatch alarm rule or can be run using any
 │   └── restore_state_machine.png
 ├── src
 │   ├── checkstatus
-│   │   ├── DBClusterStatusWaiter.py              <-- Python Waiter for checking the status of the cluster
+│   │   ├── DBClusterStatusWaiter.py              <-- Python [Waiter](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/clients.html) for checking the status of the cluster
```

Comment:
Can we modify the link to https://boto3.amazonaws.com/v1/documentation/api/latest/guide/clients.html#waiters

Reply:
@stationeros thanks. Changed. Kindly review again?

Reply:
@despot  Thanks ... can you resolve the conflicting files.

Reply:
@stationeros done.